packages/server/test/security/api.test.ts

```diff
@@ -89,63 +116,46 @@ describe('API', () => {
       }
     })
 
-    test('Should allow anyone to make a sync request', async () => {
```

Comment:
Maybe we could convert this test into `Should **not** allow everyone to make a sync request`

Reply:
I added a test for that